Library/src/CSPFoundation.cpp

```diff
@@ -382,12 +390,19 @@ const csp::common::String& CSPFoundation::GetTenant()
 
 void CSPFoundation::SetClientUserAgentInfo(const csp::ClientUserAgent& ClientUserAgentHeader)
```

Comment:
Do we still need this method if the ClientUserAgent info is being passed in via the Initialize method?

Reply:
I kept it just in case clients want to update again. But I can make private if we dont think its necessary

Reply:
It might be better to hide it tbf. I dont know why they would want to do this. What do you think?

Reply:
Yes I agree, I think we should make it private